api4/group.go

```diff
@@ -0,0 +1,450 @@
+// Copyright (c) 2018-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+package api4
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/mattermost/mattermost-server/model"
+)
+
+const (
+	groupMemberActionCreate = iota
+	groupMemberActionDelete
```

Comment:
I notice you've used `camelCase` for the constants, although we use `SNAKE_CAPS` everywhere else in the code base.

I presume this is some golang best practice? Not sure how I feel about inconsistency vs. switching to the proper golang convention.

Reply:
Yeah Golint doesn't like snakes.

Reply:
Ok fair enough.